argilla/docs/reference/argilla-server/cli.md

```diff
@@ -0,0 +1,199 @@
+# Server CLI
```

Comment:
@davidberenstein1957 @jfcalvo these docs don't belong to 2.0, the `cli` is not supported for 2.0.